common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -505,19 +521,19 @@
             };
 
             DiscussionThreadListView.prototype.getNameWidth = function(name) {
-                var test, width;
-                test = $("<div>");
-                test.css({
+                var $test, width;
+                $test = $("<div>");
+                $test.css({
                     "font-size": this.$(".forum-nav-browse-current").css('font-size'),
                     opacity: 0,
                     position: 'absolute',
                     left: -1000,
                     top: -1000
                 });
-                $("body").append(test);
-                test.html(name);
-                width = test.width();
-                test.remove();
+                $("body").append($test);
+                $test.html(edx.HtmlUtils.ensureHtml(name).toString());
```

Comment:
Is name plain text?  If so, I'd use `$test.text(name)`.  Otherwise, this is fine, or you could use `HtmlUtils.setHtml()`.